src/oidc_clients/models/client.ts

```diff
@@ -0,0 +1,25 @@
+import { IResponseType } from './responsetype';
+
+export interface IOidcClientOwner {
+  first_name: string;
+  last_name: string;
+  email: string;
+  username: string;
+}
```

Comment:
I think we already have this defined as a type, something like `IPublicUser` or something like that?

Reply:
Ah, I'll look into that. Thanks

Reply:
The user types we have in profile/models/User doesn't overlap properly with IOidcClientOwner.

Reply:
Is there too many fields or is something lacking?
`interface IOidcCliendOwner extends Partial<IPublicUser>` could perhaps be used then, to symbolize a relation between the models?

Reply:
We apparently have two serializers for displaying users, we should probably just have one: https://github.com/dotkom/onlineweb4/blob/develop/apps/authentication/serializers.py#L27-L39